### ISSUE TITLE
Futures Multiplex Socket Receives Spot Data

### ISSUE THREAD

**Issue by alexjnelson** [sammchardy/python-binance#896]
**Describe the bug**
When using the USDM futures multiplex socket for <symbol>@bookTicker, the order ID and prices are the same as those received by a spot multiplex socket. To my knowledge it is the futures socket receiving spot data, because the order IDs are consistent with the spot order IDs when I directly connected to the websocket. This does not appear to be a problem when connecting with an individual start_symbol_ticker_futures_socket, which works correctly for both USDM and COINM future.

**To Reproduce**
```
symbols = ['BTCUSDT', 'ETHUSDT']

twm = ThreadedWebsocketManager()
twm.start()

twm.start_multiplex_socket(callback=lambda msg: print('Spot:', msg), streams=[f'{s.lower()}@bookTicker' for s in symbols])
twm.start_futures_multiplex_socket(callback=lambda msg: print('Futures:', msg), streams=[f'{s.lower()}@bookTicker' for s in symbols])
```

**Expected behavior**
The futures socket output should show different order ID's and prices than the spot socket's output.

**Environment (please complete the following information):**
 - Python version: 3.8
 - Virtual Env: None
 - OS: Windows and Ubuntu (wsl): issue persists across each
 - python-binance version 1.0.11

**Comment by sammchardy** [comment]
This is fixed in v1.0.12

**Comment by alexjnelson** [comment]
Great, thanks!
